src/components/NFTPage/PreviewCard.vue

```diff
@@ -96,6 +97,26 @@
         </template>
       </ButtonV2>
 
+      <template v-if="isContentViewable">
+        <ButtonV2
+          v-for="contentUrl in contentUrls"
+          :key="contentUrl"
+          class="mt-[12px]"
+          preset="outline"
+          :text="$t(`nft_details_page_button_view_${getContentUrlType(contentUrl)}`)"
```

Comment:
preferable not to generate i18n key paths dynamically, it is problematic when removing unused paths/find & replace